### DISCUSSION TITLE
Arabic input issue

### DISCUSSION THREAD

**Discussion by Alihkhawaher** [AppGround-io/bluetooth-keyboard-and-mouse-support#299] [Bugs & Problems]
Amazing app

I added Arabic to the layout list but and switched to Arabic, still no input. English input is fine

![Screenshot_20230721_171715_Bluetooth Keyboard   Mouse](https://github.com/AppGround-io/bluetooth-keyboard-and-mouse-support/assets/7887710/05433cb0-107f-4e48-93fb-e857295d350e)
![Screenshot_20230721_171650_Bluetooth Keyboard   Mouse](https://github.com/AppGround-io/bluetooth-keyboard-and-mouse-support/assets/7887710/62ee6db2-3024-49fc-b2dd-0dcc8e17e355)

**Comment by AppGround-io** [comment]
You have to change the keyboard language on the connected device to the layout selected in the app.

If you are using Windows, here is a guide how to change the keyboard layout:

https://www.windowscentral.com/how-change-your-keyboard-layout-windows-10
